frontend/www/js/omegaup/components/problem/Tags.vue

```diff
@@ -44,59 +44,57 @@
             </tr>
           </tbody>
         </table>
-        <template v-if="!hidePrivateTags">
-          <div class="form-group">
-            <label class="font-weight-bold">{{ T.wordsPrivateTags }}</label>
-            <div class="input-group">
-              <input
-                type="text"
-                class="form-control"
-                v-model="newPrivateTag"
-                v-bind:placeholder="T.privateTagsPlaceholder"
-              />
-              <div class="input-group-append">
+        <div class="form-group">
+          <label class="font-weight-bold">{{ T.wordsPrivateTags }}</label>
+          <div class="input-group">
+            <input
+              type="text"
+              class="form-control"
+              v-model="newPrivateTag"
+              v-bind:placeholder="T.privateTagsPlaceholder"
+            />
+            <div class="input-group-append">
+              <button
+                class="btn btn-outline-primary"
+                type="button"
+                v-bind:disabled="newPrivateTag === ''"
+                v-on:click.prevent="addPrivateTag"
+              >
+                {{ T.wordsAddTag }}
+              </button>
+            </div>
+          </div>
+        </div>
+        <table class="table table-striped">
+          <thead>
+            <tr>
+              <th class="text-center" scope="col">
+                {{ T.contestEditTagName }}
+              </th>
+              <th class="text-center" scope="col">
+                {{ T.contestEditTagDelete }}
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-for="tag in selectedPrivateTags" v-bind:key="tag">
+              <td class="align-middle">
+                <a v-bind:href="`/problem/?tag[]=${tag}`">
+                  {{ tag }}
+                </a>
+              </td>
+              <td class="text-center">
                 <button
-                  class="btn btn-outline-primary"
                   type="button"
-                  v-bind:disabled="newPrivateTag === ''"
-                  v-on:click.prevent="addPrivateTag"
+                  class="btn btn-danger"
+                  v-on:click="removeTag(tag, false /* public */)"
```

Comment:
nit:

```suggestion
                  v-on:click="removeTag(tag, /*public=*/false)"
```

los comentarios con los nombres de los parámetros siempre van antes